doc/method/introduction.md

```diff
@@ -44,28 +43,31 @@ taxonomic knowledge.
 Although it is primarily a phylogenetics effort, Open Tree requires a
 reference taxonomy for each of these functions.
 
-In 1, the taxonomy is used for converting OTUs (operational
-taxonomic units, or 'tips') to a canonical form.  Supertree
-construction requires that a source tree OTU be matched with an OTU
-from another tree whenever possible.  This is a nontrivial task.  The
-approach taken is to map OTUs to the reference taxonomy, so that OTUs
-are compared by comparing the taxa to which they map.
+In 1, the taxonomy is used for converting OTUs (operational taxonomic
+units, or 'tips') to a canonical form.  Supertree construction
+requires that a source tree OTU be matched with an OTU from another
+tree whenever possible.  This is a nontrivial task because a single
+taxon can be designated as an OTU in different source trees in
+different ways.  The approach taken is to map OTUs to the reference
+taxonomy, so that OTUs in different source trees are compared by
+comparing the taxa to which they map.
 
 In 2, the groupings in the supertree are compared to those in the
 taxonomy.
 
-In 3, the taxonomy provides supplemental taxa beyond the ones
-that are in the source trees, using the outcome of step 2 to determine
-where they are to be 'grafted'.
+In 3, the taxonomy provides supplemental taxa beyond the ones that are
```

Comment:
add "only a small number of known taxa are represented in source trees, so" between "In 3" and "the taxonomy provides".